plugins/frontend/backstage-plugin-iframe/README.md

```diff
@@ -0,0 +1,73 @@
+# Iframe plugin
+
+## Features
+
+- Displays a simple iframe from a url
+
+## Plugin Setup
+
+1. If you have standalone app (you didn't clone this repo), then in the [packages/app](https://github.com/backstage/backstage/blob/master/packages/app/) directory of your backstage instance, add the plugin as a package.json dependency:
+
+```bash
+yarn add @roadiehq/backstage-plugin-iframe
+```
+
+2. Import the plugin to the [entityPage.tsx](https://github.com/backstage/backstage/blob/master/packages/app/src/components/catalog/EntityPage.tsx) source file:
+
+```ts
+// packages/app/src/components/catalog/EntityPage.tsx
+import { 
+  iframePlugin,
+  EntityIFrameCard,
+  EntityIFramePage,
+} from '@roadiehq/backstage-plugin-iframe';
+...
+
+const iframeProps = {
+  frames: [
+    {
+      src: "https://example.com"
+    }
+  ]
+}
+const serviceEntityPage = (
+  <EntityLayoutWrapper>
+    ...
+    <EntityLayout.Route 
+      path="/mycustom-iframes"
+      title="Iframes">
+        <EntityIFramePage {...iframeProps} />
```

Comment:
EntityPages should only have Content type components within them as tabs. 
Take a look here for the conventions: https://backstage.io/docs/plugins/composability#naming-patterns

Reply:
Ah ok, I see.

Thank you @Xantier